control-plane-operator/controllers/hostedcontrolplane/cvo/reconcile.go

```diff
@@ -70,6 +70,8 @@ var (
 		"0000_50_cluster-ingress-operator_02-deployment-ibm-cloud-managed.yaml",
 		"0000_70_cluster-network-operator_02_rbac.yaml",
 		"0000_70_cluster-network-operator_03_deployment-ibm-cloud-managed.yaml",
+		"0000_80_machine-config-operator_01_containerruntimeconfig.crd.yaml",
```

Comment:
@csrwng can we just stop including this in the ibm-cloud-managed profile?
and also exclude.release.openshift.io/internal-openshift-hosted for future profile consumption?